pkg/vault/test/foundry/unit/VaultExtensionUnit.t.sol

```diff
@@ -15,6 +15,10 @@ contract VaultExtensionUnitTest is BaseVaultTest {
         BaseVaultTest.setUp();
     }
 
+    function testItWorks() public pure {
+        assertTrue(true);
+    }
```

Comment:
remove?

Reply:
Ha :) Yes, I do this when debugging an error in the setup.